src/app/home/events-type-list/events-type-list.component.html

```diff
@@ -1,13 +1,10 @@
 <app-title-fades [color]="titleColor" [title]="titleText" />
 
 <ion-list lines="none" class="categories-list">
-  <ion-card
-    class="category-card all"
-    [routerLink]="['/tabs/events', { sortBy: 'all' }]"
-  >
+  <ion-card class="category-card all" (click)="redirectWithState('all')">
```

Comment:
Why so? Maybe this is what you were looking for? Then there will be no need of a new function and etc
    [routerLink]="['/tabs/events']"
    [queryParams]="{ sortBy: 'all' }"

Reply:
Maybe i find better way.